docs/security/outbound-requests.md

```diff
@@ -75,6 +75,7 @@ In addition, when the "Include statistics" option is enabled, we'll send some sp
 | The usage of step configuration options | 2020.2.5 |
 | Aggregated type, result code and duration of database and api calls | 2020.2.14 |
 | Counts of specific kinds of events that occur internally, e.g. usage of deprecated code paths, errors | 2020.3.0 |
+| Structured Configuration Variable usage across projects including how many steps have Structured Configuration Variables enabled and what file extensions are being used | 2020.4.0
```

Comment:
Looks like you might need a final `|` at the end of the line.

Reply:
Thanks :)

Reply:
FYI, I've seen it work both ways. I always assumed the final pipe was required, but apparently you don't have to include them :)